### ISSUE TITLE
Incorrect column size

### ISSUE THREAD

**Issue by eliquious** [johnotander/rework-flex-grid#4]
Hey,

I noticed a possible issue with the small column widths. I've got 2 columns each at 50% for all 3 screen sizes (`.grid-row-col-6-12--{sm/md/lg}`). There's a title on the left and a button on the right. When collapsed to < 32rem (512px I think), they both stay at 100% width. Where as they should stay at 50% each.

Here's a [gist](https://gist.github.com/a1401f4fa9d4702135a6.git).

The md and lg sizes work fine. It seems to be the `flex-basis: 100%` rule causing the issue. Any ideas?

**Comment by johnotander** [comment]
Thanks for creating this issue.

Yeah, I think that currently the `--sm` modifier is set to work between `32rem` and `48rem` by default. In order to get the grid to work on > `32rem` currently it would also require the `.grid-row-col-6-12` class without a modifier. However, as I reflect more on it, it seems broken.

It's a poor configuration for the media queries right now. I suspect they should really be: anything less than `32rem` is `--sm`, `32rem` to `48rem` is `--md`, and greater than `48rem` is `--lg`. Or, the other option is to keep the current queries, and add `--xs` modifier.

What are your thoughts @eliquious ?

**Comment by eliquious** [comment]
Yeah, I guess I misunderstood the media queries. What about adding an `--xs` modifier for < `32rem` and keeping the others where they are at? Some tablets are at `64rem` so moving the `--lg` to `48rem` might lump some tablets and desktops together.

**Comment by johnotander** [comment]
>  I guess I misunderstood the media queries. 

That means I have a documentation bug, : ).

> What about adding an --xs modifier for < 32rem and keeping the others where they are at?

I think that's the way to go, for sure.
